corporate/migrations/0030_alter_zulipsponsorshiprequest_requested_plan.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 4.2.8 on 2023-12-19 11:38
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("corporate", "0029_session_tier"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="zulipsponsorshiprequest",
+            name="requested_plan",
+            field=models.CharField(
+                choices=[
+                    ("", "UNSPECIFIED"),
+                    ("Community", "COMMUNITY"),
+                    ("Basic", "BASIC"),
+                    ("Business", "BUSINESS"),
+                ],
+                default="",
+                max_length=50,
+            ),
+        ),
+    ]
```

Comment:
I guess this should be squashed into the commit adding the new plan.

Reply:
Done.